oshi-core/src/main/java/oshi/hardware/platform/unix/openbsd/OpenBsdHWDiskStore.java

```diff
@@ -62,6 +57,68 @@ private OpenBsdHWDiskStore(String name, String model, String serial, long size)
         super(name, model, serial, size);
     }
 
+    /**
+     * Gets the disks on this machine.
+     *
+     * @return an {@code UnmodifiableList} of {@link HWDiskStore} objects representing the disks
+     */
+    public static List<HWDiskStore> getDisks() {
+        List<OpenBsdHWDiskStore> diskList = new ArrayList<>();
+
+        // Get list of disks from sysctl
+        // hw.disknames=sd0:2cf69345d371cd82,cd0:,sd1:
+        String[] devices = OpenBsdSysctlUtil.sysctl("hw.disknames", "").split(",");
+        OpenBsdHWDiskStore store;
+        for (String device : devices) {
+            // get partitions using disklabel command (requires root)
+            Pair<Map<String, String>, List<HWPartition>> diskdata = Disklabel
+                .getDiskParams((device.split(":")[0]));
+            if (diskdata.getA().isEmpty()) {
+                // something must have gone wrong, probably no elevated/root permission
+                store = new OpenBsdHWDiskStore((device.split(":")[0]), Constants.UNKNOWN, Constants.UNKNOWN, 0L);
```

Comment:
Do we have a backup non-root source of at least some partition info, particularly for mounted partitions?  I think `df` or `iostat` probably can at least help here.